packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -1387,11 +1370,9 @@ export class PHP implements Disposable {
 		 */
 		const newFs = this[__private__dont__use].FS;
 		for (const path of oldRootLevelPaths) {
-			if (
-				path &&
-				getNodeType(oldFS, path) === 'memfs' &&
-				['memfs', 'missing'].includes(getNodeType(newFs, path))
-			) {
+			// The /request directory holds per-request state that is isolated to a
+			// single PHP instance. Let's not copy it.
+			if (path && path !== '/request') {
```

Comment:
@adamziel, great catch! Thank you! We don't want to be duplicating request state.